File: server/dotnet/HostedTokenization/CreateHostedTokenizationBasicDto.cs
namespace OnlinePayments.Sdk.Example;

/// <summary>
/// Basic Dto for hosted tokenization used for testing purposes
/// </summary>
public class CreateHostedTokenizationBasicDto
{
    public decimal Amount { get; set; }

    public string? Currency { get; set; }

    public string? HostedTokenizationId { get; set; }

}

File: server/java/src/main/java/com/worldlinesolutions/onlinepayments/example/hostedtokenization/CreateHostedTokenizationBasicDto.java
package com.worldlinesolutions.onlinepayments.example.hostedtokenization;

import java.math.BigDecimal;

/**
 * Basic Dto for hosted tokenization used for testing purposes
 */
public class CreateHostedTokenizationBasicDto {

    private BigDecimal amount;

    private String currency;

    private String hostedTokenizationId;

    public BigDecimal getAmount() {
        return amount;
    }

    public void setAmount(BigDecimal amount) {
        this.amount = amount;
    }

    public String getCurrency() {
        return currency;
    }

    public void setCurrency(String currency) {
        this.currency = currency;
    }

    public String getHostedTokenizationId() {
        return hostedTokenizationId;
    }

    public void setHostedTokenizationId(String hostedTokenizationId) {
        this.hostedTokenizationId = hostedTokenizationId;
    }
}
